balancer-labs/frontend-v2
# Hotfix: Sanctions check via Balancer API

# Description

Re-add the sanctions check (removed in: #2028), performing it using a Lambda running on the Balancer API instead. 

PR to add the Lambda API endpoint: https://github.com/balancer-labs/pools-api/pull/22

## Type of change


## How should this be tested?

Perform same tests as in #2017. Ensure that normal wallets connect fine and sanctioned wallets are blocked

## Checklist:
